src/Toggle.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
+  export let checked: boolean;
+</script>
+
+<style>
+  .toggle input[type="checkbox"] {
+    display: none;
+  }
+
+  .toggle label {
+    background-color: var(--color-background-1);
+    border: 1px solid var(--color-foreground-6);
+    border-radius: 50px;
+    cursor: pointer;
+    display: block;
+    position: relative;
+    transition: transform ease-in-out 0.2s;
+    width: 40px;
+    height: 1.5rem;
+  }
+
+  .toggle label::after {
+    background-color: var(--color-foreground-6);
+    border-radius: 50%;
+    content: " ";
+    cursor: pointer;
+    display: inline-block;
+    position: absolute;
+    left: 3px;
+    top: 3px;
+    transition: transform ease-in-out 0.2s;
+    width: 1rem;
+    height: 1rem;
+  }
+
+  .toggle input[type="checkbox"]:checked ~ label {
+    background-color: var(--color-background-1);
+    border-color: var(--color-foreground-6);
+  }
+
+  .toggle input[type="checkbox"]:checked ~ label::after {
+    background-color: var(--color-foreground-6);
+    transform: translateX(15px);
+  }
+</style>
+
+<div class="toggle">
+  <input type="checkbox" bind:checked on:change id="toggle" />
```

Comment:
Nice, I like that we're using an actual checkbox for this. 👍

Reply:
If we'll have multiple `<Toggle>` components on the same page, there will be an `id` collision. Could you look into making that more robust?

Reply:
Okay took me some time, but I think I found an interesting solution, avoiding the usage of some additional npm package and to avoid hopefully any id collision
48babb94

Reply:
Browser compatibility is better than I imagined
https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID#browser_compatibility